tsconfig.packages.json

```diff
@@ -8,7 +8,8 @@
         "rootDir": "src",
         "strict": true,
         "strictPropertyInitialization": false,
-        "target": "es2021",
+        "//": "The target should be kept in sync with `bin/build_packages.ts` file.",
+        "target": "es2022",
```

Comment:
Can we add polyfills for Object.hasOwn() and Array.at() ?

Reply:
Ok yeah that seems fair (even 97% for both in real usage it seems) ..

.. but after this one, please can we say we do not make changes like this in minor/patch version ? 😆 🙏